tests/allow_relation/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
```

Comment:
Please try to reuse existing models rather than adding a new test app. `multiple_databases` might be a good candidate.

Reply:
I'll simplify the tests and get them working in that test app.